src/nft/components/profile/list/SelectMarketplacesDropdown.tsx

```diff
@@ -0,0 +1,172 @@
+import { useOnClickOutside } from 'hooks/useOnClickOutside'
+import { Column, Row } from 'nft/components/Flex'
+import { ChevronUpIcon } from 'nft/components/icons'
+import { Checkbox } from 'nft/components/layout/Checkbox'
+import { buttonTextMedium, caption } from 'nft/css/common.css'
+import { ListingMarket } from 'nft/types'
+import { ListingMarkets } from 'nft/utils/listNfts'
+import { Dispatch, FormEvent, useMemo, useReducer, useRef } from 'react'
+import styled from 'styled-components/macro'
+import { ThemedText } from 'theme'
+import { Z_INDEX } from 'theme/zIndex'
+
+const HeaderButtonWrap = styled(Row)`
+  padding: 12px;
+  border-radius: 12px;
+  width: 220px;
+  justify-content: space-between;
+  background: ${({ theme }) => theme.backgroundModule};
+  cursor: pointer;
+  &:hover {
+    background-color: ${({ theme }) => theme.backgroundInteractive};
+  }
+`
+
+const HeaderButtonContentWrapper = styled.div`
+  display: flex;
+  flex-direction: row;
+`
+
+const MarketIcon = styled.img<{ index: number; totalSelected: number }>`
+  height: 20px;
+  width: 20px;
+  margin-right: 8px;
+  border: 1px solid;
+  border-color: ${({ theme }) => theme.backgroundInteractive};
+  border-radius: 4px;
+  z-index: ${({ index, totalSelected }) => totalSelected - index};
+  margin-left: ${({ index }) => `${index === 0 ? 0 : -18}px`};
+`
+
+const Chevron = styled(ChevronUpIcon)<{ isOpen: boolean }>`
+  height: 20px;
+  width: 20px;
+  transition: ${({
+    theme: {
+      transition: { duration },
+    },
+  }) => `${duration.fast} transform`};
+  transform: ${({ isOpen }) => `rotate(${isOpen ? 0 : 180}deg)`};
+`
+
+const ModalWrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  position: relative;
+`
+
+const DropdownWrapper = styled(Column)<{ isOpen: boolean }>`
+  padding: 16px 0px;
+  background-color: ${({ theme }) => theme.backgroundModule};
+  display: ${({ isOpen }) => (isOpen ? 'flex' : 'none')};
+  position: absolute;
+  top: 52px;
+  width: 220px;
+  border-radius: 12px;
+  gap: 12px;
+  z-index: ${Z_INDEX.modalBackdrop};
+`
```

Comment:
I'm a little confused about the intended styles here, especially considering the other two dropdowns.

Reply:
whichever style we're going with, I think the styles should be pulled out so they can be shared/consistent

Reply:
which other dropdowns?

Reply:
the days and set price by are supposed to be similar whereas the marketplace one is unique

this is an interim as design is trying to standardize dropdowns across the app